File: igrant_org_sdk/src/main/java/io/igrant/igrant_org_sdk/listener/UsagePurposeClickListener.java
package io.igrant.igrant_org_sdk.listener;

import io.igrant.igrant_org_sdk.models.Organizations.PurposeConsent;

public interface UsagePurposeClickListener {

    void onItemClick(PurposeConsent consent);

    void onSetStatus(PurposeConsent consent, Boolean isChecked);
}

File: privacy-dashboard/src/main/java/com/github/privacydashboard/modules/home/UsagePurposeClickListener.kt
package com.github.privacydashboard.modules.home

import com.github.privacydashboard.models.PurposeConsent

interface UsagePurposeClickListener {
    fun onItemClick(consent: PurposeConsent?)
    fun onSetStatus(consent: PurposeConsent?, isChecked: Boolean?)
}